### PR TITLE
Create actions menu for project#show

### PR DESCRIPTION
**Pull request by dmarcoux**
On large devices:
actions-large

On small devices:
actions-small

### CODE REVIEW

**Comment by dmarcoux** [comment]
[Contextual Links](https://trello.com/c/bLctMUJL/1336-contextual-links)

**Comment by DavidKang** [comment]
@dmarcoux, we have a side link(patchinfo presente), that does the same as 'show patchinfo'. So maybe se could remove one of them. What do you think about that?

**Comment by dmarcoux** [comment]
> @dmarcoux, we have a side link(patchinfo presente), that does the same as 'show patchinfo'. So maybe se could remove one of them. What do you think about that?

Good point. I will remove the `Show Patchinfo` link and it also allows me to get rid of the CSS for properly aligning the icon stacks since it's the only icon stack.

**Comment by dmarcoux** [comment]
I wonder however if the side links should be in the actions menu. What do you think?

**Comment by hennevogel** [comment]
> I wonder however if the side links should be in the actions menu.

Those links are more information/status then links. I would leave them alone.

File: src/api/app/views/webui/project/responsive_ux/_actions.html.haml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      = render partial: 'webui/project/responsive_ux/actions/delete_project', locals: { project: project }
+    - elsif !project.is_locked?
+      = render partial: 'webui/project/responsive_ux/actions/request_role_addition_and_deletion', locals: { project: project }
+  - elsif project.key_info.present?
+    = render partial: 'webui/project/responsive_ux/actions/gpg_key_ssl_certificate', locals: { project: project }
+
+  %li.nav-item
+    = link_to(image_templates_path, class: 'nav-link') do
+      %i.fas.fa-compact-disc.fa-lg.mr-2
+      New Image
```
Comment (krauselukas) [inline src/api/app/views/webui/project/responsive_ux/_actions.html.haml]: @dmarcoux New Image currently appears in the "Actions on this page" section of the actions menu. To me that's more a general action. But not sure if its necessary at this point to introduce an additional area in the action menu for such things. I'm okay with leaving it here

Comment (dmarcoux) [reply src/api/app/views/webui/project/responsive_ux/_actions.html.haml]: I was also a bit divided on this one and in the end, I think it's okay to leave it with the others.

**Pull request merged**